packages/core/package.json

```diff
@@ -39,7 +39,7 @@
     "prepublishOnly": "pnpm build"
   },
   "dependencies": {
-    "dedent": "^1.5.1",
+    "strip-indent": "^3.0.0",
```

Comment:
Replaced `dedent` with `strip-indent` to address similar issues to https://github.com/dmnd/dedent/issues/20